testing/cpm/cpm_find-and-find_package/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ include(${rapids-cmake-dir}/cpm/find.cmake)
 cmake_minimum_required(VERSION 3.26.4)
 project(rapids-test-project LANGUAGES CXX)
 
-
+message(STATUS "CMAKE_FIND_PACKAGE_REDIRECTS_DIR: ${CMAKE_FIND_PACKAGE_REDIRECTS_DIR}")
```

Comment:
Is this an intentional change, or just for testing?

Reply:
testing and needs to be removed

Reply:
```suggestion
```